trustmanager/filestore.go

```diff
@@ -61,7 +75,52 @@ func (f *fileStore) Add(name string, data []byte) error {
 	return ioutil.WriteFile(filePath, data, f.perms)
 }
 
-// Remove removes a file identified by name
+// AddEncrypted writes encrypted data to a file with a given name, given a key
+func (f *fileStore) AddEncrypted(name string, data []byte, passphrase string) error {
```

Comment:
Its a bit confusing b/c the GCM lib's notion of `data` is the associated data for AEAD mode.